.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,119 @@
+name: 🐞 bug fix
+description: "file a bug in noop services"
+title: "[Bug]: "
+labels: ["bug"]
+assignees:
+  - noop-cloud/noop
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Thanks for taking the time to fill out this bug report!
+  - type: markdown
+    attributes:
+      value: |
+        - Please review open issues labeled as `bugs` to ensure your issue is not a duplicate.
+        - For general questions, ideas and concerns that are not related to a bug, visit our [discussions page](https://github.com/noop-cloud/supreme-garbanzo/discussions)
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
+  - type: dropdown
+    id: affected-os
+    attributes:
+      label: Operating System
+      description: What Operating System(s) have you experienced this bug on?
+      multiple: true
+      options:
+        - Windows
+        - MacOS
```

Comment:
If someone is using the web version of the console we can't guarantee that they'll only be using a Window or MacOS based machine.

Reply:
so text field? or does the OS not matter to us on issue intake?

Reply:
Text field. OS is a very import intake detail. I just don't think it would be realistic for us to maintain a definitive list of all operating systems someone is using while accessing Noop.

Right now the options are only `Windows` and `MacOS`. What would someone select if they weren't using either of those OSes?